### PR TITLE
html-tables

### PR DESCRIPTION
**Pull request by John-bess4**
Screen Shot 2023-05-16 at 7 30 25 AM

### CODE REVIEW

File: html-tables/index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title></title>
```
Comment (shawnkost) [inline html-tables/index.html:4]: The name of the exercise needs to go here!

File: html-tables/index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    <h1>Student Grade Table</h1>
+    <table>
+      <thead>
+        <tr>
+          <th>Student</th>
+          <th scope="col">Course</th>
+          <th scope="col">Grade</th>
+        </tr>
+        <tr>
+          <th scope="row">Grace Hopper</th>
```
Comment (shawnkost) [inline html-tables/index.html:16]: This should be a `td` element

File: html-tables/index.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+          <td>Computer Science</td>
+          <td>100</td>
+        </tr>
+
+        <tr>
+          <th scope="row">Alan Turing</th>
```
Comment (shawnkost) [inline html-tables/index.html:22]: This should be a `td` element

File: html-tables/index.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+          <td>Mathematics</td>
+          <td>100</td>
+        </tr>
+
+        <tr>
+          <th scope="row">Douglas Crockford</th>
```
Comment (shawnkost) [inline html-tables/index.html:28]: This should be a `td` element

File: html-tables/index.html [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+          <td>JSON</td>
+          <td>98</td>
+        </tr>
+
+        <tr>
+          <th scope="row">Tim Berners-Lee</th>
```
Comment (shawnkost) [inline html-tables/index.html:34]: This should be a `td` element

File: html-tables/index.html [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+          <td>HTML</td>
+          <td>9001</td>
+        </tr>
+
+        <tr>
+          <th scope="row">Linus Torvalds</th>
```
Comment (shawnkost) [inline html-tables/index.html:40]: This should be a `td` element

**Review by shawnkost — CHANGES_REQUESTED** [review]
Some changes requested here John!

**Comment by John-bess4** [comment]
Screen Shot 2023-05-28 at 4 48 55 PM


Fixed the TH and TD issues and also fixed the title issue as well.

**Pull request merged**